### PR TITLE
Added documentation for Ceph-Dashboard enablement

### PR DESCRIPTION
**Pull request by UtkarshBhatthere**
Added docs for enabling ceph-dashboard and integrating it with Prometheus and Alertmanager.

### CODE REVIEW

**Comment by nobuto-m** [comment]
What about Grafana integration? Also, what about HA of Ceph Dashboard? Do we have a single and highly available entry point to access the dashboard?

**Comment by UtkarshBhatthere** [comment]
> What about Grafana integration? Also, what about HA of Ceph Dashboard? Do we have a single and highly available entry point to access the dashboard?

Hey @nobuto-m ! I am happy to see that you are excited about MicroCeph. We don't have ceph-dashboard HA available or in plans right now. I will raise this in ceph team discussions so that we seed it into our active backlog. About grafana, I will go ahead and add that documentation to the MicroCeph docs as well.

File: docs/tutorial/enable_dashboard.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Enable Ceph-Dashboard
+=====================
+
+Introduction
+------------
+
+Ceph-Dashboard is a built-in web based application for monitoring and management of your Ceph cluster. This makes it easy for administer Ceph clusters for users who prefer WebUI over CLI. It is implemented as a ceph-mgr module and thus is served by the active ceph-mgr daemon.
```
Comment (sabaini) [inline docs/tutorial/enable_dashboard.rst:7]: Typos, should be "easy to administer"

File: docs/tutorial/enable_dashboard.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+..  code-block:: none
+
+    ceph dashboard set-prometheus-api-ssl-verify False
+    ceph dashboard set-alertmanager-api-ssl-verify False
+
+Let's check if the dashboard is receiving alerts!
+
+.. figure:: assets/dashboard_monitoring.jpeg
+
+  Ceph Dashboard: Monitoring
```
Comment (sabaini) [inline docs/tutorial/enable_dashboard.rst:71]: Suggest some closing phrase, e.g.: 

> As an example, this shows details of a CephHealthWarning

**Review by sabaini — CHANGES_REQUESTED** [review]
Lgtm in general, found a typo and added a suggestion

File: docs/tutorial/enable_dashboard.rst (hunk 2 above)
Comment (UtkarshBhatthere) [reply docs/tutorial/enable_dashboard.rst]: Ack

**Pull request merged**